src/pydata_sphinx_theme/theme/pydata_sphinx_theme/_templates/version-switcher.html

```diff
@@ -39,6 +39,10 @@
 (function () {
     $.getJSON("{{ theme_switcher.get('json_url') }}", function(data, textStatus, jqXHR) {
         const currentFilePath = "{{ pagename }}.html";
+        let btn = document.getElementById("version_switcher_button");
+        // Set empty strings for active button by default so we can demo in developer mode
```

Comment:
```suggestion
        // Set empty strings so that these attributes exist and can be used in CSS selectors
```

Reply:
after reading your comment I think this is actually a better description

Reply:
Ah sure 😃!